models/intermediate/hubspot/int_rag_hubspot__deal_comment_document.sql

```diff
@@ -126,7 +126,7 @@ comment_markdowns as (
         comment_time,
         cast(
             {{ dbt.concat([ 
-                "'Email subject:'", "email_subject", "'\\n'",
+                "'Email subject:'", "title", "'\\n'",
```

Comment:
```suggestion
                "'Email subject:'", "email_title", "'\\n'",
```
In line with the above suggestions